.github/workflows/release.yml

```diff
@@ -73,10 +73,36 @@ jobs:
         with:
           name: nph-${{ matrix.target.name }}.tar.gz
           path: nph-${{ matrix.target.name }}.tar.gz
+  
+  vsix:
+    name: Build .vsix file
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install deps
+        run: |
+          cd vscode-nph
+          yarn      
+      
+      - name: Build .vsix extension
+        id: buildext
+        uses: HaaLeo/publish-vscode-extension@v1
+        with:
+          pat: "dummyVal"
```

Comment:
?

Reply:
When using the `dryRun` value to just build the extension, the [docs](https://github.com/marketplace/actions/publish-vs-code-extension) for this github action state the following:
```
When using this option set the pat option to a stub value.
```

`pat` would only have a valid `p`ersonal `a`ccess `t`oken when you intend to publish the extension to one of the market places, and then, it would look like this:
```
- name: Publish to Open VSX Registry
        uses: HaaLeo/publish-vscode-extension@v1
        with:
          pat: ${{ secrets.OPEN_VSX_TOKEN }}
- name: Publish to Visual Studio Marketplace
        uses: HaaLeo/publish-vscode-extension@v1
        with:
          pat: ${{ secrets.VS_MARKETPLACE_TOKEN }}
          registryUrl: https://marketplace.visualstudio.com
```